src/ansys/fluent/core/launcher/container_launcher.py

```diff
@@ -59,6 +63,13 @@ def __init__(
         ----------
         mode : FluentMode
             Launch mode of Fluent to point to a specific session type.
+        exposure : FluentExposure
+            Sets the exposure of Fluent. The possible values are the values of the
+            ``FluentExposure`` enum.
+        graphics_driver : FluentWindowsGraphicsDriver or FluentLinuxGraphicsDriver
+            Sets the graphics driver of Fluent. The possible values are the values of
+            the ``FluentWindowsGraphicsDriver`` enum in Windows or the values of
+            the ``FluentLinuxGraphicsDriver`` enum in Linux.
         product_version : str, optional
             Select an installed version of ANSYS. The string must be in a format like
             ``"23.2.0"`` (for 2023 R2) matching the documented version format in the
```

Comment:
```suggestion
            Exposure of Fluent. Options are the values of the
            ``FluentExposure`` enum.
        graphics_driver : FluentWindowsGraphicsDriver or FluentLinuxGraphicsDriver
            Graphics driver of Fluent. Options are the values of
            the ``FluentWindowsGraphicsDriver`` enum in Windows or the values of
            the ``FluentLinuxGraphicsDriver`` enum in Linux.
        product_version : str, optional
            Installed version of Ansys. The string must be in a format like
            ``"23.2.0"`` (for 2023 R2), matching the documented version format in the
```
Descriptions for parameters use be in noun form and not start with verbs.

Reply:
Updated, thanks